api_tests/entities/update_claims.test.coffee

```diff
@@ -66,3 +66,20 @@ describe 'entities:update-claims', ->
     .catch undesiredErr(done)
 
     return
+
+  it 'should accept rapid updates on the same entity', (done)->
+    authorsUris = [ 'wd:Q192214', 'wd:Q206685', 'wd:Q281411', 'wd:Q312835', 'wd:Q309945' ]
```

Comment:
just to be sure it really works ^^ what is the downside of having that many uris? (out of some additional 500ms to run ;))

Reply:
dude... dont kill the `api_tests`... calm down... dude take it easy, it will work fine enough... i understand its to make sure.. i understand... and then they say its not safe i know i know... but none of this is *really* necessary

Reply:
addressed by c2d8e30